client/src/hooks/Input/useTextarea.ts

```diff
@@ -1,8 +1,10 @@
 import debounce from 'lodash/debounce';
 import React, { useEffect, useRef, useCallback } from 'react';
-import { EModelEndpoint } from 'librechat-data-provider';
-import type { TEndpointOption } from 'librechat-data-provider';
+import { useRecoilState } from 'recoil';
+import store from '~/store';
```

Comment:
Please bring back what you removed and organize imports by length:

First, library/package imports, from shortest to longest,
Then, types from longest to shortest
Last, local project files from longest to shortest

Reply:
solved in 2832434ba157810b771043d729b5ff60346ce278